APITestingFramework/Core/RestHelper.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Text.Json;
+using System.Threading.Tasks;
+using Models;
+using RestSharp;
+using RestSharp.Authenticators;
+using RestSharp.Serializers.Json;
+
+namespace Core;
+
+public class RestHelper
+{
+    private readonly RestClient client;
+
+    public RestHelper(string uri)
+    {
+        client = new RestClient(uri);
+    }
+
+    public void AddDefaultHeader(string Key, string value)
+    {
+        client.AddDefaultHeader(Key, value);
+    }
+
+    public RestResponse Get(string url)
+    {
+        var request = new RestRequest(url, Method.Get);
```

Comment:
try to avoid the user of var, always use the variable type.

Reply:
Done